packages/react/src/NavList/NavList.docs.json

```
@@ -136,6 +136,37 @@
           "type": "React.RefObject<HTMLElement>"
         }
       ]
+    },
+    {
+      "name": "NavList.TrailingAction",
+      "props": [
+        {
+          "name": "as",
+          "type": "a | button",
+          "defaultValue": "button",
+          "required": false,
+          "description": "HTML element to render as."
+        },
+        {
+          "name": "label",
+          "type": "string",
+          "defaultValue": "",
+          "required": true,
+          "description": "Acccessible name for the control."
+        },
+        {
+          "name": "icon",
+          "type": "string",
+          "defaultValue": "",
+          "required": true,
+          "description": "Octicon to pass into IconButton. When this is not set, TrailingAction renders as a `Button` instead of an `IconButton`."
+        },
+        {
+          "name": "ref",
+          "type": "React.RefObject<HTMLAnchorElement>",
```

Should this be `href`? We might need to change this in `ActionList.docs.json` too

```suggestion
          "name": "href",
          "type": "string",
```

---

Nice catch!!